src/main/java/org/itxtech/synapseapi/SynapseAPI.java

```diff
@@ -45,6 +45,17 @@ public void onLoad() {
 
     @Override
     public void onEnable() {
+        this.getServer().getScheduler().scheduleRepeatingTask(new cn.nukkit.scheduler.Task() {
+            @Override
+            public void onRun(int i) {
+                for (Player player : Server.getInstance().getOnlinePlayers().values()) {
+                    player.setHealth(player.getHealth());
```

Comment:
Isn't food bar the only thing we have problems with?

Reply:
nop, sometime heal too

Reply:
How about experience?